core/capability-negotiation-3.2.md

```diff
@@ -42,7 +42,8 @@ If no other arguments are present, version 3.1 is assumed.
 ### Capabilities with values
 
 Servers MAY specify additional data for each capability using the
-`<name>[=<value>]` format.
+`<name>[=<value>]` format for `CAP LS` and `CAP NEW`.  `CAP ACK` and
+`CAP NAK` replies MUST NOT contain any values.
```

Comment:
Good catch, as this could become a maintenance burden should other non-value-accepting replies be added.  In order to reinforce that *only* `CAP LS` and `CAP NEW` may contain values without explicitly listing, would this be acceptable?
```md
Servers MAY specify additional data for each capability using the
`<name>[=<value>]` format for `CAP LS` and `CAP NEW`.  Other replies
MUST NOT contain any values.
```

If it's good, I'll update the Errata, too.